src/components/portfolio/SmallCheckout/SmallCheckout.tsx

```diff
@@ -219,6 +230,24 @@ const SmallCheckout = ({
                 </Field>
               </Box>
             </Flex>
+            {contributionValue > 0 && taxInPercent && taxInPercent > 0 && (
+              <Text size="smLowNormal" mt="2">
+                {formatMessage(
+                  { id: 'portfolio.smallCheckout.price.taxIncluded' },
+                  {
+                    number: formatNumber(
+                      contributionValue +
+                        (contributionValue * taxInPercent) / 100,
+                      {
+                        style: 'currency',
+                        currency,
+                        maximumFractionDigits: 0,
```

Comment:
```suggestion
                        maximumFractionDigits: 2,
```

Since it's a price I would use 2.